pkgs/development/python-modules/typedunits/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  stdenv,
+  cython,
+  setuptools,
+  attrs,
+  numpy,
+  protobuf,
+  pyparsing,
+  pytestCheckHook,
+}:
+
+buildPythonPackage {
+  pname = "typedunits";
+  version = "0-unstable-2025-05-09";
```

Comment:
https://pypi.org/project/typedunits/#history lists some versions. Maybe we should ask upstream to tag them.

Reply:
I asked: https://github.com/quantumlib/TypedUnits/issues/67

Reply:
No response. How would you like to proceed?

Reply:
I guess we should use `fetchPypi` until there are tags.

Reply:
ping @sarahec

Reply:
I gave fetchPypi a try and it's a mess. They don't have a source distro there, just platform- (and python-) specific [wheels](https://pypi.org/project/typedunits/#files).

I have two options:
1. Go back to github, pin on the commit matching the last wheel release date, and give the developer time to respond, or
2. Write a n-way switch to handle {darwin,linux}x{x86_64,aarch64}x{py312,py313} and a matching shell script to pull all the wheels and calculate their hashes.

Thoughts?

Reply:
1 sounds good

Reply:
Done. Running review